common-ui-compose/src/main/java/app/tivi/common/compose/ambients.kt

```diff
@@ -16,20 +16,7 @@
 
 package app.tivi.common.compose
 
-import androidx.compose.Composable
-import androidx.compose.Providers
 import androidx.compose.staticAmbientOf
 import app.tivi.util.TiviDateFormatter
 
 val TiviDateFormatterAmbient = staticAmbientOf<TiviDateFormatter>()
```

Comment:
a whole file for an ambient? Consider creating a `ProvideDateFormatter` here similar to insets and renaming the file.

Reply:
I think eng don't want to use wrap ambients too much. `ProvideInsets` is a bit special because it needs to observe a live data, so wrapping it up makes sense. This is a very simple static ambient though.

Reply:
/cc @adamp